sdk/cdsclient/interface.go

```diff
@@ -147,6 +147,15 @@ type HatcheryClient interface {
 	HatcheryCount(wfNodeRunID int64) (int64, error)
 }
 
+// BroadcastClient expose all function for CDS Broadcasts
+type BroadcastClient interface {
+	Broadcasts() ([]sdk.Broadcast, error)
```

Comment:
Broadcasts and BroadcastByLevel could be one function:
`Broadcasts(filteredLevels ...string)`